Fix regular expression that searches for CD-HIT version number.
Based on: https://github.com/sanger-pathogens/Roary/commit/3badecef2446f36a73b191d7df3814fccec90d3d
diff -ru Roary-3.13.0.orig/lib/Bio/Roary/External/CheckTools.pm Roary-3.13.0/lib/Bio/Roary/External/CheckTools.pm
--- Roary-3.13.0.orig/lib/Bio/Roary/External/CheckTools.pm	2019-11-06 11:55:40.000000000 +0100
+++ Roary-3.13.0/lib/Bio/Roary/External/CheckTools.pm	2020-06-08 13:26:31.000000000 +0200
@@ -97,7 +97,7 @@
     },
     'cd-hit' => {
         GETVER => "cd-hit -h | grep 'CD-HIT version'",
-        REGEXP => qr/version\s+($BIDEC)i/,
+        REGEXP => qr/version\s+($BIDEC)/i,
         MINVER => "4.6",
     }
 );
